package.json

```diff
@@ -52,11 +52,10 @@
     "trailingComma": "all"
   },
   "scripts": {
-    "build": "npm run tsc",
```

Comment:
Why remove the build script? This is pretty conventional including on travis-ci or heroku. Was that required to run on `now`?

Reply:
It is not necessary for ZEIT Now and it looks like it is just an alias for the `tsc` script so I removed it.

Reply:
I reverted, thanks